lib/output.js

```diff
@@ -67,7 +67,8 @@ function OutputStream(options) {
         screw_ie8        : false,
         preamble         : null,
         quote_style      : 0,
-        keep_quoted_props: false
+        keep_quoted_props: false,
+        ecma             : 5
```

Comment:
Actually, since the harmony branch is ecma6 by default perhaps you want `ecma5_strings` in this case?